testdata/serviceconfig.yml

```diff
@@ -30,19 +30,10 @@ rules:
           response_code: response.code | 200
   - kind: access-logs
     params:
-      logName: "access_log"
+      logName: access_log
       log:
-        logFormat: 1 # COMMON
-        templateExpressions:
```

Comment:
can we keep the labels?  i still think we want the structured logs, if possible -- even if the config is ugly as sin.